### PR TITLE
Fixing some cosmetic issues with the output voltage measurement examp…

### PR DESCRIPTION
**Pull request by DelpireNI**
…le measui.

- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Updated the measui for the Output Voltage Measurement example to fix cosmetic issues. We should now be more consistent with the other examples' UIs.

The Input and Output pin controls are now outside the grey boxes, and at the top like other examples.
The labels over the grey boxes have been renamed to Source configuration and Measurement configuration.

image

### Why should this Pull Request be merged?

Consistency with other examples.
(#419)

### What testing has been done?

Example still runs. This is just a cosmetic change.

### CODE REVIEW

**Comment by DelpireNI** [comment]
Other examples move `Measurement type` outside of the configuration box as well. I left it in because it allows the grey boxes to align and I believe this looks a bit better. The measurement type does feel like a "measurement configuration" to me.

**Comment by DelpireNI** [comment]
It feels like there is too much margin at the top. I left it close to the existing UI's margin, but I'm happy to shift things up if others agree.

**Comment by bkeryan** [comment]
> It feels like there is too much margin at the top. I left it close to the existing UI's margin, but I'm happy to shift things up if others agree.

I agree, the top/bottom margin are too big.

Also, the grey boxes have inconsistent left/bottom/right margins. (I'm ignoring top because of the label.)

**Comment by DelpireNI** [comment]
> > It feels like there is too much margin at the top. I left it close to the existing UI's margin, but I'm happy to shift things up if others agree.
> 
> I agree, the top/bottom margins are inconsistent and too big.
> 
> Also, the grey boxes have inconsistent left/bottom/right margins. (I'm ignoring top because of the label.)
> 
> Also the spacing between columns.

Okay I have made the margins more consistent and reduced the height.

**Review by dixonjoel — APPROVED** [review]

File: examples/output_voltage_measurement/OutputVoltageMeasurement.measui
Comment (bkeryan) [inline examples/output_voltage_measurement/OutputVoltageMeasurement.measui:1]: Please reduce the spacing between the **Source configuration** and **Measurement configuration** boxes.

Comment (bkeryan) [inline examples/output_voltage_measurement/OutputVoltageMeasurement.measui:1]: Consider setting the pin control width to be the same as the grey box below it. We haven't done this consistently, but we have done this in some examples (e.g. dcpower, scope).

**Review by bkeryan — CHANGES_REQUESTED** [review]

File: examples/output_voltage_measurement/OutputVoltageMeasurement.measui
Comment (DelpireNI) [reply examples/output_voltage_measurement/OutputVoltageMeasurement.measui:1]: Okay I have reduced the space.

Comment (DelpireNI) [reply examples/output_voltage_measurement/OutputVoltageMeasurement.measui:1]: Fixed.

Comment (DelpireNI) [reply examples/output_voltage_measurement/OutputVoltageMeasurement.measui:1]: Let me know if you want even less space. :)

**Review by bkeryan — APPROVED** [review]

File: examples/output_voltage_measurement/OutputVoltageMeasurement.measui
Comment (bkeryan) [reply examples/output_voltage_measurement/OutputVoltageMeasurement.measui:1]: I would have set it the same as the margin but it's not a big deal, at least it looks deliberate now

**Pull request merged**